urth/dashboard/converter/__init__.py

```diff
@@ -187,6 +197,12 @@ def add_urth_widgets(output_path, notebook_file):
         # urth widgets not installed so skip
         return
 
+    # Check if any of the cells contain widgets, if not we do not to copy the bower_components
+    notebook = nbformat.read(notebook_file, 4)
+    cell_widget_bools = map(lambda cell: cell_uses_widgets(cell.get('source')), notebook.cells)
```

Comment:
Can use any() here to shortcut. Don't need to loop through the remainder if any one cell uses import.